model.py

```diff
@@ -419,6 +419,16 @@ def library_stage(self, value):
     def pls_id(self):
         return ConfigurationSetting.for_library(Library.PLS_ID, self)
 
+    def number_of_patrons(self, _db):
+        # This is only meaningful if the library is in production.
+        if not self.in_production:
+            return '0'
```

Comment:
This should return a number in both cases since the method is called "number". If it needs to be converted into a string that should happen outside.